WireAPI/Sources/WireAPI/Private/Extensions/JSONDecoder+Default.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension JSONDecoder {
+
+    /// The default decoder to use when parsing http response payloads.
+
+    static var defaultDecoder: JSONDecoder {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .custom { decoder in
+            let container = try decoder.singleValueContainer()
+            let rawDate = try container.decode(String.self)
+
+            guard let date = Date(transportString: rawDate) else {
+                throw DecodingError.dataCorruptedError(
+                    in: container,
+                    debugDescription: "Expected date string to be ISO8601-formatted with fractional seconds"
+                )
+            }
+
+            return date
+        }
```

Comment:
Same question as above - shouldn't we go with `.dateDecodingStrategy = .iso8601` by default?

Reply:
I don't think so because we need the fractional seconds don't we?